indico/modules/events/timetable/operations.py

```diff
@@ -118,3 +121,57 @@ def fit_session_block_entry(entry, log=True):
         entry.event_new.log(EventLogRealm.management, EventLogKind.change, 'Timetable',
                             "Session block fitted to contents", session.user,
                             data={'Session block': entry.session_block.full_title})
+
+
+def move_timetable_entry(entry, parent=None, day=None):
+    """Move the `entry` to another session or top-level timetable
+
+    :param entry: `TimetableEntry` to be moved
+    :param parent: If specified then the entry will be set as a child
+                         of parent
+    :param day: If specified then the entry will be moved to the
+                        top-level timetable on this day
+    """
+    if bool(parent) + bool(day) != 1:
+        raise Exception("Wrong number of arguments")
```

Comment:
That's the one you get when you pass the wrong amount of arguments as far as I know.